src/main/java/com/github/fge/jsonpatch/operation/OmitOperation.java

```diff
@@ -0,0 +1,27 @@
+package com.github.fge.jsonpatch.operation;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.fge.jackson.jsonpointer.JsonPointer;
+import com.github.fge.jsonpatch.operation.policy.PathMissingPolicy;
+
+/**
+ * Extended JSON Patch {@code omit} operation.
+ *
+ * The operation will remove the {@code value} at {@code path} if it exists.
+ * It will do nothing if the actual value at {@code path} is not equal to {@code value}.
+ * It will throw a "no such path" error if there is no value at {@code path}.
+ */
+public final class OmitOperation extends OmitOperationBase
+{
+    public static final String OPERATION_NAME = "omit";
+
+    @JsonCreator
+    public OmitOperation(@JsonProperty("path") final JsonPointer path,
+                         @JsonProperty("value") final JsonNode value)
+    {
+        super(OPERATION_NAME, path, value, PathMissingPolicy.THROW);
```

Comment:
Yeah, I've been thinking about how to do that.  I bet you can write a custom deserializer for JsonPatchOperation that will parse the "op", figure out which action and policies, and compose an actual JsonPatchOperation instance composed of the action and policy.  This would obviate the need to create all these subclasses of JsonPatchOperation.

The Action implementation would contain the parameters for the action, e.g. path and value.  We may still need to use inheritance for serialization of the patch operation, but at least the inheritance would be limited to the major Actions, and not the combinations of Policies.